### DISCUSSION TITLE
Deleted Transaction than can not re-import from CSV

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#5347] [General]
After deleting the transactions and trying to use data-importer:develop to re-import it generate a duplicated message.

Am I doing something wrong or missing some steps ?

Thanks
Van Der

**Comment by JC5** [comment]
Firefly III will also check deleted transactions when checking for duplicates.

https://docs.firefly-iii.org/data-importer/help/faq/#why-cant-i-import-duplicate-transactions
